lasagne/layers/pool.py

```diff
@@ -755,3 +756,161 @@ def get_output_shape_for(self, input_shape):
 
     def get_output_for(self, input, **kwargs):
         return self.pool_function(input.flatten(3), axis=2)
+
+
+def pooling_fixed_output_size(inputs, output_size, pooling_op=T.max):
```

Comment:
Not sure about the name... I'll see if I can find a better one. It's good to move this into a separate function, though.